custom_components/localtuya/cover.py

```diff
@@ -139,15 +139,15 @@ def __get_status(self):
 #                    return None
                     raise ConnectionError("Failed to update status .")
 
-    def set_status(self, state, switchid):
-        #_LOGGER.info("running def set_status from cover")
+    def set_dps(self, state, switchid):
```

Comment:
Seems to be that `switchid` is an old copy-paste mistake, maybe we should change that to index or something?

Reply:
Fully support this change.  `index` or `dpsindex`.

Reply:
OK for both, done